requirements-app.txt

```diff
@@ -1,64 +1,64 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
 
-apig-wsgi==2.13.0
+apig-wsgi==2.15.0
 boto==2.49.0
-cffi==1.14.5
+cffi==1.15.1
 celery[sqs]==5.0.5
 docopt==0.6.2
-environs==9.3.2 # pyup: <9.3.3 # marshmallow v3 throws errors
+environs==9.5.0 # pyup: <9.3.3 # marshmallow v3 throws errors
 fido2==0.9.1
 Flask-Bcrypt==0.7.1
 flask-marshmallow==0.14.0
 Flask-Migrate==2.7.0
 git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
-Flask==2.0.3
+Flask==2.2.2
 click-datetime==0.2
 eventlet==0.30.2 # currently 0.31.0+ breaks gunicorn. Test the docker image if upgrading!
 gunicorn==20.1.0
 iso8601==0.1.14
-idna==2.8 # pinned to align with test moto dependency requirements
+idna==2.10 # pinned to align with test moto dependency requirements
 jsonschema==3.2.0
-marshmallow-sqlalchemy==0.23.1 # pyup: <0.24.0 # marshmallow v3 throws errors
+marshmallow-sqlalchemy==0.28.1 # pyup: <0.24.0 # marshmallow v3 throws errors
 marshmallow==2.21.0 # pyup: <3 # v3 throws errors
 python-magic==0.4.22
-psycopg2-binary==2.8.6
-PyJWT==2.1.0
-pytz==2021.1
+psycopg2-binary==2.9.5
+PyJWT==2.6.0
+pytz==2021.3
 PyYAML==5.4.1
 SQLAlchemy==1.4.37
-sentry-sdk[flask]==1.0.0
-cachelib==0.1.1
+sentry-sdk[flask]==1.10.1
+cachelib==0.9.0
 
-newrelic==6.2.0.156
-notifications-python-client==6.0.2
-python-dotenv==0.17.1
+newrelic==6.10.0.165
+notifications-python-client==6.4.1
```

Comment:
Ok I didn't expect that, it's quite cool renovate keeps us up to date with latest. We'll need to be careful though with this bump.